lib/webpacker/dev_server_runner.rb

```diff
@@ -45,6 +45,7 @@ def detect_port!
 
       def execute_cmd
         env = Webpacker::Compiler.env
+        env.merge("WEBPACKER_CONFIG" => @webpacker_config)
```

Comment:
@justin808 don't think this would mutate the env? Maybe simply assign instead of merge.

```rb
env["WEBPACKER_CONFIG"]  =  @webpacker_config)
```

Reply:
@gauravtiwari fixed!